toolchain/check/testdata/function/builtin/call_from_operator.carbon

```diff
@@ -24,53 +14,6 @@ impl i32 as Core.Add {
 
 var arr: [i32; 1 + 2] = (3, 4, 3 + 4);
```

Comment:
Given that this is a low-level test for calling a builtin via an operator interface, I think we should make it a no-prelude test.

Reply:
Done